DGN/wwwroot/js/Statistics.js

```diff
@@ -0,0 +1,177 @@
+﻿// First statistic: articles_per_category
+// set the dimensions and margins of the graph
+var width = 450
+height = 450
+margin = 40
+
+// The radius of the pieplot is half the width or half the height (smallest one), subtract a bit of margin.
+var radius = Math.min(width, height) / 2 - margin
+
+// append the svg_articles_per_category object to the div called 'articles_per_category'
+var svg_articles_per_category = d3.select("#articles_per_category")
+    .append("svg")
+    .attr("width", width)
+    .attr("height", height)
+    .append("g")
+    .attr("transform", "translate(" + width / 2 + "," + height / 2 + ")");
+
+// Get the data
+var ArticlesByCategorty = {}
+var ArticlesCount = 0;
+
+$.ajax({
+    type: "GET",
+    url: "/Articles/GetArticlesByCategory",
+    success: function (dataJson) {
+        dataJson.forEach(currCategory => {
+            ArticlesCount += currCategory.articlesInCategory;
+            ArticlesByCategorty[currCategory.categoryName] = currCategory.articlesInCategory;
+        })
+    },
+    error: function () {
+        $("#alert-body").html('ERROR');
+    }
+}).done(function() { 
+    DrawPiechart(ArticlesByCategorty)
+});
+
+function DrawPiechart(data) {
```

Comment:
Very good!!! but maybe we should do the text not on the pie itself but as a template with square of color and near it the category name and percentage.
I think it will be prettier and may not be to complicated to do.

Reply:
This is a different issue to make it prettier :)